indigo_app/views/tasks.py

```diff
@@ -164,13 +168,33 @@ class TaskEditView(TaskViewBase, UpdateView):
 
     def form_valid(self, form):
         task = self.object
+        old_workflows = [wf.id for wf in task.workflows.all()]
         task.updated_by_user = self.request.user
-        action.send(self.request.user, verb='updated', action_object=task,
-                    place_code=task.place.place_code)
         task.workflows = form.cleaned_data.get('workflows')
 
-        # TODO: send action signal saying 'User added|removed Task to|from Workflow,
-        #  and don't send  generic 'updated' signal if this was the only change
+        # action signals
+        # first, was something changed other than workflows?
+        if form.changed_data:
```

Comment:
I think `form.changed_data` would also include updated workflows?

Reply:
if only the workflows are changed `changed_data` is empty, I know because I initially looked there for changes to workflows

Reply:
Cool.